modules/actions/workflows.go

```diff
@@ -29,6 +29,14 @@ func init() {
 	}
 }
 
+func IsWorkFlow(path string) bool {
```

Comment:
This function name seems inconsistent. Other function names are `ListWorkflows` `DetectWorkflows` : Work[f]low